src/components/profile/editProfile/ChangePassword.jsx

```diff
@@ -118,6 +122,22 @@ const ChangePassword = props => {
   );
 };
 
+ChangePassword.defaultProps = {
+  pristine: () => {},
+  reset: false,
+  submitting: () => {},
+  error: "",
+  handleSubmit: () => {},
+};
+
+ChangePassword.propTypes = {
+  pristine: PropTypes.func,
+  reset: PropTypes.bool,
+  submitting: PropTypes.func,
+  error: PropTypes.string,
+  handleSubmit: PropTypes.func,
+};
```

Comment:
reset isn't bool (func), submitting and pristine aren't func (bool)

Reply:
Done